packages/profiling-node/src/utils.ts

```diff
@@ -158,6 +179,82 @@ function createProfilePayload(
   return profile;
 }
 
+/**
+ * Create a profile chunk from raw thread profile
+ * @param {RawThreadCpuProfile} cpuProfile
+ * @param {options}
+ * @returns {Profile}
+ */
+function createProfileChunkPayload(
+  client: Client,
+  cpuProfile: RawChunkCpuProfile,
+  {
+    release,
+    environment,
+    start_timestamp,
+    trace_id,
+    profiler_id,
+    chunk_id,
+  }: {
+    release: string;
+    environment: string;
+    start_timestamp: number;
+    trace_id: string | undefined;
+    chunk_id: string;
+    profiler_id: string;
+  },
+): ProfileChunk {
+  // Log a warning if the profile has an invalid traceId (should be uuidv4).
+  // All profiles and transactions are rejected if this is the case and we want to
+  // warn users that this is happening if they enable debug flag
+  if (trace_id && trace_id.length !== 32) {
+    DEBUG_BUILD && logger.log(`[Profiling] Invalid traceId: ${trace_id} on profiled event`);
```

Comment:
shouldn't we avoid sending profiles with an invalid trace ID in general?

I guess we don't have any controls in relay over this 😬